src/sonic-device-data/Makefile

```diff
@@ -13,6 +13,7 @@ $(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
 	# Create a new dir and copy all ONIE-platform-string-named dirs into it
 	mkdir ./device
 	cp -r ../../../device/*/* ./device/
+	find ./device/ -type l -exec bash -c 'test -e {} && (original=$$(readlink -f {}) && rm -rf {} && cp -rf -L $$original {} ) ' \;
```

Comment:
Is it possible to use one line for both cp and resolving?
```
cp -r -H ../../../device/*/* ./device/
```

Reply:
Yes. Since we don't need anymore "hw-managment" links to "/etc/mlx/msn*" which could not be resolved in build server file system. So now it is removed and can links can be resolved during "cp".